script/bootstrap.py

```diff
@@ -92,6 +95,18 @@ def update_node_modules(dirname, env=None):
       execute_stdout([NPM, 'install'], env)
 
 
+def update_crashpad():
+  gclient = os.path.join(VENDOR_DIR, 'depot_tools', 'gclient.py')
```

Comment:
We should make it as a submodule instead of relying external tool to clone it, so we can clearly track the commit of crashpad we are using.